python/ray/scripts/scripts.py

```diff
@@ -177,11 +177,18 @@ def cli(logging_level, logging_format):
     is_flag=True,
     default=False,
     help="do not redirect non-worker stdout and stderr to files")
+@click.option(
+    "--plasma-store-socket-name",
+    is_flag=True,
+    default=False,
+    help="manually specify the name of the socket of the plasma store"
+)
```

Comment:
The point of exposing this is so that you can reuse an existing plasma store? Seems reasonable, but do you have a use case in mind?

Reply:
That's connected with #2687.

For us this is a way to share data objects computed with Ray with UI tools which can then expose data values which were computed without having to go through some other serialization or memory copying.